### ISSUE TITLE
[<Library component: Models|Core|etc...>] 

### ISSUE THREAD

**Issue by HAUNGTOFU** [Nixtla/neuralforecast#1180]
### Description

model2=[TiDE(h=horizon,
                  input_size=12,
                  loss=MAE(),
                  scaler_type='minmax',
                  learning_rate=1e-3,
                  max_steps=100,
                  val_check_steps=100,
                  early_stop_patience_steps=4,
                  futr_exog_list=['vacation_num'],
                  batch_size=32)]
model3=[TiDE(h=horizon,
                  input_size=12,
                  loss=MAE(),
                  scaler_type='minmax',
                  learning_rate=1e-3,
                  max_steps=100,
                  val_check_steps=100,
                  early_stop_patience_steps=4,
                  futr_exog_list=['timeinformation'],
                  batch_size=32)]
I added two covariates separately, but why are the prediction results exactly the same?
